src/plugins/score-plugin-scenario/Scenario/Document/Tempo/TempoView.hpp

```diff
@@ -56,8 +57,51 @@ class TempoPresenter final
       QObject* parent)
       : CurveProcessPresenter{style, layer, view, context, parent}
   {
+    // only disable the curve when using the first 2 inlets
+    for (int i = 0; i < 2; i++)
+    {
+      QObject::connect(
+          layer.inlets()[i],
+          &Process::Inlet::addressChanged,
+          this,
+          &TempoPresenter::disableWithAddress);
+
+      QObject::connect(
+          layer.inlets()[i],
+          &Process::Inlet::cablesChanged,
+          this,
+          &TempoPresenter::disableWithCables);
+    }
   }
 
   void setFullView() override { m_curve.setBoundedMove(false); }
+
+private:
+  void disableWithAddress(const State::AddressAccessor& address)
+  {
+    disableIfNeeded();
+  }
+
+  void disableWithCables() { disableIfNeeded(); }
+
+  void disableIfNeeded()
+  {
+    bool should_disable{false};
+
+    // only disable the curve when using the first 2 inlets
+    for (int i = 0; i < 2; i++)
+    {
+      const auto inlet{m_process.inlets()[i]};
+
+      if (inlet->address().toString_unsafe() != ":/"
```

Comment:
hmm, there should be a better way to do that check, I'll look around in score (and add a method "isValid" or something to State::Address if that's not the case)

Reply:
I can add it in AddressAccessor if you wish